src/serializers/postman/Serializer.js

```diff
@@ -280,11 +281,17 @@ export default class PostmanSerializer extends BaseSerializer {
                 if (_body.get('value') instanceof JSONSchemaReference) {
                     let ref = _body.get('value')
                     if (this._isInlineRef(ref)) {
-                        data += JSON.stringify(
-                            _body.getJSONSchema(false),
-                            null,
-                            '  '
-                        )
+                        let schema = _body.getJSONSchema(false)
+                        if (schema.type === 'string' && schema.default) {
+                            data += _body.generate()
+                        }
+                        else {
+                            data += JSON.stringify(
+                                _body.getJSONSchema(false),
+                                null,
+                                '  '
+                            )
+                        }
```

Comment:
This bit of code is getting increasingly complex and I feel like I should maybe flesh it out a bit better in a future PR.